src/main/java/io/stargate/sgv2/jsonapi/service/schema/tables/ApiSupportDef.java

```diff
@@ -86,47 +118,57 @@ record Support(boolean createTable, boolean insert, boolean read, boolean filter
    * @param read If non-null, will match objects where the read value is the same.
    * @param filter If non-null, will match objects where the filter value is the same.
    */
-  record Matcher(Boolean createTable, Boolean insert, Boolean read, Boolean filter)
+  record Matcher(
```

Comment:
yeah, this was not my best idea maybe. Thanks for updating we may want to kill this later.